### ISSUE TITLE
Notification not working 

### ISSUE THREAD

**Issue by Richspiral** [j-kaltes/Juggluco#208]
![Screenshot_20241008_111333_One UI Home.jpg](https://github.com/user-attachments/assets/dc47d279-eca9-4bf7-ad1f-4465cd7bac30)

Since the recent update the notification is not working properly. As you can see in the screenshot the font is in the background and not properly visible.

This did happen with the previous version but I could tap on it, go thru to the app and it would adjust to be visible. That is no longer possible. 

The only way to get it to work now is to go into my notification settings and turn the notification on and off again.

For info my phone switches to dark mode in the evening and light mode during the day and it is when these modes are switched that the notification becomes much less visible.

Thank you.

**Comment by j-kaltes** [comment]
Yes, when you switch from dark mode to light mode, the notification is momentarily invisible. The background changed color, but the notification is not rewritten. But when a new glucose value arrives it becomes visible again.

**Comment by Richspiral** [comment]
Thanks for your prompt response. 🙏

The notification is not becoming visible again. It's nearly 2pm here in the UK and day mode has been active for many hours with my numbers rising. 

It's still invisible.

Even with the old version i had to click thru to the app to get visibility. That method is no longer working.

**Comment by j-kaltes** [comment]
Restart Juggluco.

**Comment by Richspiral** [comment]
Yep that works if I force stop the app and start it again, as does the method I outlined in my original post of turning on and off the notification.

**Comment by j-kaltes** [comment]
Now I also see it. It will be fixed in the next version.

**Comment by Richspiral** [comment]
Thanks for your efforts and work as always 👍
